src/Build/PublicAPI/netstandard/PublicAPI.Unshipped.txt

```diff
@@ -14,6 +14,7 @@ Microsoft.Build.Experimental.MSBuildClientExitType.ServerBusy = 1 -> Microsoft.B
 Microsoft.Build.Experimental.MSBuildClientExitType.Success = 0 -> Microsoft.Build.Experimental.MSBuildClientExitType
 Microsoft.Build.Experimental.MSBuildClientExitType.UnableToConnect = 2 -> Microsoft.Build.Experimental.MSBuildClientExitType
 Microsoft.Build.Experimental.MSBuildClientExitType.Unexpected = 4 -> Microsoft.Build.Experimental.MSBuildClientExitType
+Microsoft.Build.Experimental.MSBuildClientExitType.UnknownServerState = 5 -> Microsoft.Build.Experimental.MSBuildClientExitType
```

Comment:
Weird that this is creeping into your PR. Seems like it should have been added in #8024

Reply:
Yeah, this wasn't intentional. Will let the people involved in #8024 fix it.